linux_os/guide/system/accounts/accounts-restrictions/password_storage/no_empty_passwords/ansible/ubuntu2004.yml

```diff
@@ -0,0 +1,10 @@
+# platform = multi_platform_ubuntu
+# reboot = false
+# strategy = configure
+# complexity = low
+# disruption = medium
+- name: Ensure 'nullok' is removed in /etc/pam.d/common-password
```

Comment:
Please, ensure the [Project Style Guides](https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#ansible)